scripts/generate_config.py

```
@@ -22,8 +22,22 @@
+import os
 import shutil
 import sys
+from typing import Any, get_args
+
+# src を sys.path に追加して context_store をインポート可能にする
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "src"))
+
+try:
+    from context_store.config import Settings
+except ImportError:
+    # インポート失敗時のフォールバック（スタンドアロン実行用）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)